components/Card/Card.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import styled from 'styled-components';
+import Colors from '../Colors';
+import {
+  Header,
+  HeaderText,
+  Title,
+  Description,
+  Media,
+  Thumbnail,
+  Content,
+  Footer,
+} from './sub-components';
+
+const CardWrapper = styled.article`
+  background-color: ${Colors.WHITE};
+  border-radius: 8px;
+  box-shadow: 0 2px 6px 0 ${Colors.SHADOW[50]};
+  color: ${Colors.BLACK[700]};
```

Comment:
I guess this color should inherit from body according to @ddsilva, right?

Reply:
Right!
Changed.